plugins/database_api/api.cpp

```diff
@@ -22,6 +23,8 @@
 
 namespace golos { namespace plugins { namespace database_api {
 
+    using namespace golos::plugins::witness_api;
```

Comment:
For which cases are used this namespace?

Reply:
It is needed to resolve witness_api_object from get_witnesses. i moved witness_api_object to other witness_api. But i guess we dont need whole namespace here, its probably beter to just resolve witness_api_object in places.

Reply:
Yes i will remove it